lib/components/Keyboard/KeyboardInput/KeyboardAccessoryView.tsx

```diff
@@ -202,13 +181,15 @@ class KeyboardAccessoryView extends Component<KeyboardAccessoryViewProps> {
       onItemSelected,
       onRequestShowKeyboard,
       useSafeArea,
+      scrollBehavior = KeyboardAccessoryView.iosScrollBehaviors.FIXED_OFFSET,
+      iOSScrollBehavior,
       ...others
     } = this.props;
 
     return (
       <KeyboardTrackingView
         {...others}
-        scrollBehavior={this.getIOSTrackingScrollBehavior()}
+        scrollBehavior={IsIOS ? iOSScrollBehavior || scrollBehavior : undefined}
```

Comment:
Not sure why this was conditioned, we pass other iOS only props to Android, do we really need this?